formbuilder/config.js

```diff
@@ -195,5 +208,37 @@ export default {
       },
       formData: {}
     }
+  ],
+  fieldSets: [
+    {
+      id: "address",
+      icon: "leaf",
+      label: "Postal address",
+      jsonSchema: {
+        type: "object",
+        title: "",
```

Comment:
Yes we do, still to ensure required keys are defined and so we can propagate values retrieved from the form properties edition form, as explained in a previous comment.